api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -9,34 +9,98 @@
  */
 package org.openmrs.notification.db.hibernate;
 
+import org.junit.Assert;
 import org.junit.Before;
+
+import java.util.List;
+
+import org.hibernate.SessionFactory;
 import org.junit.Test;
+import org.openmrs.User;
 import org.openmrs.api.context.Context;
-import org.openmrs.notification.AlertService;
+import org.openmrs.notification.Alert;
 import org.openmrs.test.BaseContextSensitiveTest;
 
 /**
  * This class tests the hibernate alert data access. TODO Consider changing this and all subsequent
  * tests to use dbunit
  */
 public class HibernateAlertDAOTest extends BaseContextSensitiveTest {
+	private HibernateAlertDAO hibernateAlertDAO;
 	
 	@Before
 	public void runBeforeEachTest() {
+		Context.openSession();
 		authenticate();
+		hibernateAlertDAO = new HibernateAlertDAO();
+		hibernateAlertDAO.setSessionFactory((SessionFactory) applicationContext.getBean("sessionFactory"));
 	}
 	
+	/**
+	 * Creates and Returns alert with Id 1234
+	 * 
+	 * @return Alert object
+	 */
+
+	private Alert createAlert() {
+		Alert alert = new Alert(1234);
+		alert.setText("Testing the hibernate DAO");
+		return alert;
+
+	}
+
 	/**
 	 * Test that you can get alerts
 	 * 
 	 * @throws Exception
 	 */
 	@Test
-	public void shouldGetAlerts() {
-		
-		AlertService as = Context.getAlertService();
-		//System.out.println(as.getAllAlerts());
-		
+	public void getAlert_shouldReturnAlert() throws Exception {
```

Comment:
Even this?, if so try reducing the space be

Reply:
ii think the space must be fine,the section in red simply means i removed that content and it has to be reflected that way @sherrif10